src/bin/add/args.rs

```diff
@@ -106,6 +109,22 @@ impl Args {
 
         Ok(vec![dependency])
     }
+
+    fn get_upgrade_prefix(&self) -> &'static str {
```

Comment:
Since this is not really performance critical (and IIRC we are talking
about a Boolean which is Copy anyway), I'd probably just `.clone()` the
flag 😄

Wayne Nilsen <notifications@github.com> schrieb am Fr., 1. Juli 2016 um
12:29:

> In src/bin/add/args.rs
> <https://github.com/killercup/cargo-edit/pull/89#discussion_r69280441>:
>
> > @@ -106,6 +109,22 @@ impl Args {
> >
> >          Ok(vec![dependency])
> >      }
> > +
> > +    fn get_upgrade_prefix(&self) -> &'static str {
>
> I also think it has something to do with the fact that the map function
> takes ownership of self which seems like a bad thing.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/killercup/cargo-edit/pull/89/files/954fa58bf3efbc162342438a922dc55d42579666#r69280441>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABOX78jo36xzt_8w0cBloSHgXvl0IGDks5qROv0gaJpZM4IzonS>
> .
>